src/components/Carousel/index.js

```diff
@@ -138,7 +138,9 @@ const Carousel = () => {
           </View>
         </View>
       ) : (
+        // $FlowFixMe
```

Comment:
What's wrong with line 142?

Reply:
the whole component has some weird error I cannot seem to fix

```
Cannot create View element because in type argument Yield [1] of the return value of property children.@@iterator:
 • Either React.Element [2] is incompatible with React.Portal [3].
 • Or property @@iterator is missing in React.Element [2] but exists in $Iterable [4].

     src/components/Carousel/index.js
     138│           </View>
     139│         </View>
     140│       ) : (
     141│         <View style={{ height: 100 }}>
     142│           {/* $FlowFixMe */}
 [2] 143│           <Swiper
     144│             style={styles.scrollView}
     145│             autoplay
     146│             autoplayTimeout={5}
     147│             showsButtons={false}
     148│             dotStyle={[styles.bullet, { backgroundColor: colors.fog }]}
     149│             activeDotStyle={[
     150│               styles.bullet,
     151│               { backgroundColor: colors.fog, opacity: 1 },
     152│             ]}
     153│           >
     154│             {slides.map(({ id, Component }) => (
     155│               <Component key={id} />
     156│             ))}
     157│           </Swiper>
     158│           <TouchableOpacity
     159│             style={styles.dismissCarousel}
     160│             hitSlop={hitSlop}
     161│             onPress={onDismiss}
     162│           >
     163│             <IconClose color={colors.smoke} size={16} />
     164│           </TouchableOpacity>
     165│         </View>
     166│       )}
     167│     </Card>
     168│   );

     /private/tmp/flow/flowlib_1e3d3ad8/core.js
 [1] 539│ interface $Iterator<+Yield,+Return,-Next> {

     /private/tmp/flow/flowlib_1e3d3ad8/react.js
 [3]  19│   | React$Portal
 [4]  20│   | Iterable<?React$Node>;
```

Reply:
This is a Swiper issue actually i'm not sure why and how to fix it but you can ignore it i guess :shrug:

Reply:
Yep